Xamarin.Forms.Platform.Android/Renderers/ScrollViewContainer.cs

```diff
@@ -23,6 +25,11 @@ public View ChildView
 
 				RemoveAllViews();
 
+				if (_childView is Layout layout1)
+				{
+					layout1.LayoutChanged -= OnChildLayoutChanged;
```

Comment:
I think we might need to unsubscribe this on Dispose.

Reply:
Excellent point.

Reply:
@rmarinho Fixed.